src/lib/isAbaRouting.js

```diff
@@ -0,0 +1,20 @@
+import assertString from './util/assertString';
+
+// http://www.brainjar.com/js/validation/
+const isRoutingReg = /^[0-9]{9}$/;
```

Comment:
Hi Panogiotis,

You are right about this. I have just double-checked [their latest article](https://www.aba.com/news-research/research-analysis/routing-number-policy-procedures). I added a separate regex to filter out reserved routing numbers, so that it can be easily modified if reserved series is brought into assigned in the future.

Thanks for your review!